groups/stack/module-ecs-services/officer-filing-api-task-definition.tmpl

```diff
@@ -0,0 +1,97 @@
+[
+    {
+        "environment": [
+            { "name": "PORT", "value": "${eric_port}" },
+            { "name": "ACCOUNT_API_URL", "value": "https://${account_subdomain_prefix}${external_top_level_domain}" },
+            { "name": "DEVELOPER_HUB_URL", "value": "https://developer${external_top_level_domain}" },
+            { "name": "CACHE_URL", "value": "${eric_cache_url}" },
+            { "name": "CACHE_MAX_CONNECTIONS", "value": "${eric_cache_max_connections}" },
+            { "name": "CACHE_MAX_IDLE", "value": "${eric_cache_max_idle}" },
+            { "name": "CACHE_IDLE_TIMEOUT", "value": "${eric_cache_idle_timeout}" },
+            { "name": "CACHE_TTL", "value": "${eric_cache_ttl}" },
+            { "name": "FLUSH_INTERVAL", "value": "${eric_flush_interval}" },
+            { "name": "GRACEFUL_SHUTDOWN_PERIOD", "value": "${eric_graceful_shutdown_period}" },
+            { "name": "DEFAULT_RATE_LIMIT", "value": "${eric_default_rate_limit}" },
+            { "name": "DEFAULT_RATE_WINDOW", "value": "${eric_default_rate_limit_window}" },
+            { "name": "PROXY_TARGET_URLS", "value": "http://${service_name}:${officer_filing_api_proxy_port}" },
+            { "name": "PROXY_BYPASS_PATHS", "value": "_single_domain_+/officers/healthcheck" },
+            { "name": "MODE", "value": "web" },
+            { "name": "LOGLEVEL", "value": "${log_level}" }
+        ],
+        "name": "eric",
+        "image": "${docker_registry}/eric:${eric_version}",
+        "cpu": 1,
+        "memory": 512,
+        "portMappings": [{
+            "containerPort": ${eric_port},
+            "hostPort": 0,
+            "protocol": "tcp"
+        }],
+       "links": [
+         "${service_name}"
+       ],
+        "logConfiguration": {
+            "logDriver": "awslogs",
+            "options": {
+                "awslogs-create-group": "true",
+                "awslogs-region": "${aws_region}",
+                "awslogs-group": "/ecs/${name_prefix}/eric",
+                "awslogs-stream-prefix": "ecs"
```

Comment:
Having ECS auto-create the log group means we are unable to easily manage the retention of logs. Please consider using a `aws_cloudwatch_log_group` resource to define and create the log group with a suitable retention period.